config/vmdb.tmpl.yml

```diff
@@ -238,6 +238,9 @@ smtp:
 snapshots:
   :create_free_percent: 100
   :remove_free_percent: 100
+ssl:
+  :ssl_ca_file:
+  :ssl_ca_path:
```

Comment:
this is actually for provider refresh, so for talking with APIs using ssl with peer verification, will be probably used for AMPQ too.

Right, we've designed it as one root CA for whole ManageIQ, talking to @mcornea. But if it should be per provider we can change it. Honestly,  I don't know what it should be. :-)